source/views/building-hours/stateful-list.js

```diff
@@ -92,19 +117,21 @@ export class BuildingHoursView extends React.Component<Props, State> {
       buildings = defaultData.data
     }
     this.setState(() => ({
-      buildings: groupBuildings(buildings),
+      buildings: groupBuildings(buildings, this.props.favoriteBuildings),
       now: moment.tz(CENTRAL_TZ),
     }))
   }
 
   render() {
+    this.fetchData()
```

Comment:
Why did you add this call here?

Reply:
Without it, the favorites are not added in the re-render of stateful-list, because groupBuildings is not called.